client/src/views/files/FoldersPage.vue

```diff
@@ -95,23 +95,23 @@
               v-if="selectedFilesCount === 0"
             >
               {{
-                $t(
-                  'FoldersPage.itemCount',
-                  { count: folders.entriesCount() + files.entriesCount() },
-                  folders.entriesCount() + files.entriesCount(),
-                )
+                $msTranslate({
+                  key: 'FoldersPage.itemCount',
+                  data: { count: folders.entriesCount() + files.entriesCount() },
+                  count: folders.entriesCount() + files.entriesCount(),
+                })
               }}
             </ion-text>
             <ion-text
               class="body item-selected"
               v-if="selectedFilesCount > 0"
             >
-              {{ $t('FoldersPage.itemSelectedCount', { count: selectedFilesCount }, selectedFilesCount) }}
+              {{ $msTranslate({ key: 'FoldersPage.itemSelectedCount', data: { count: selectedFilesCount }, count: selectedFilesCount }) }}
```

Comment:
why selectedFilesCount is present 2 times?
Is count really useful? Is data not enough?